<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#startButton{
	background-color: rgb(255, 221, 109);
	border-radius: 20px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="componentWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>438</width>
      <height>710</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="lineDesign">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>301</width>
       <height>741</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Line Design.svg</pixmap>
     </property>
    </widget>
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>257</width>
       <height>180</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="gameTitle">
        <property name="font">
         <font>
          <family>Porter Sans Block</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Guess</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gameTitle_2">
        <property name="font">
         <font>
          <family>Porter Sans Block</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="text">
         <string>The</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gameTitle_3">
        <property name="font">
         <font>
          <family>Porter Sans Block</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>420</y>
       <width>180</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Porter Sans Block</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="randonNumbers">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>420</y>
       <width>481</width>
       <height>361</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Random Numbers.png</pixmap>
     </property>
    </widget>
    <zorder>randonNumbers</zorder>
    <zorder>lineDesign</zorder>
    <zorder>titleWidget</zorder>
    <zorder>startButton</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
